docs/guides/templates/links.adoc

```diff
@@ -1,3 +1,3 @@
-<#macro server id>link:{links_server_${id}_url}[{links_server_${id}_name}]</#macro>
-<#macro operator id>link:{links_operator_${id}_url}[{links_operator_${id}_name}]</#macro>
-<#macro ha id>link:{links_high-availability_${id}_url}[{links_high-availability_${id}_name}]</#macro>
\ No newline at end of file
+<#macro server id anchor="">link:{links_server_${id}_url}<#if anchor != "">#${anchor}</#if>[{links_server_${id}_name}]</#macro>
+<#macro operator id anchor="">link:{links_operator_${id}_url}<#if anchor != "">#${anchor}</#if>[{links_operator_${id}_name}]</#macro>
+<#macro ha id anchor="">link:{links_high-availability_${id}_url}<#if anchor != "">#${anchor}</#if>[{links_high-availability_${id}_name}]</#macro>
```

Comment:
This enables anchor links as those were previously not rendered correctly, and are used for the first time in this PR.